### PR TITLE
Fix for the Bureau page on Android 4.1

### PR DESCRIPTION
**Pull request by sebworks**
Fix for the Bureau page on Android 4.1. The Mega Menu is currently breaking because of a null handling error in the assign utility function on `the-bureau` page. 

## Testing

- Visit `http://localhost:8000/about-us/the-bureau/` on an Android device via Sauce Labs. (I'm still mad that Sauce labs doesn't have Blackberry).

## Review

- @KimberlyMunoz 
- @anselmbradford 
- @jimmynotjim 

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/unprocessed/js/modules/util/assign.js [hunk 1]
```diff
@@ -28,11 +28,12 @@ function _isPlainObject( object ) {
 */
 function assign( destination ) {
   destination = destination || {};
-
+  var hasOwnProp;
   for ( var i = 1; i < arguments.length; i++ ) {
     var source = arguments[i] || {};
+    hasOwnProp = Object.hasOwnProperty.bind( source );
```
Comment (anselmbradford) [inline cfgov/unprocessed/js/modules/util/assign.js]: Will this need to use fnBind?

Comment (sebworks) [inline cfgov/unprocessed/js/modules/util/assign.js]: yeah, let me make that update.

File: cfgov/unprocessed/js/modules/util/fnBind.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* Function.Bind polyfill.
+*
+* @access private
+* @function fnBind
+* @param {function} fn - a function you want to change `this` reference to
+* @param {object} context - the `this` you want to call the function with
+* @returns {function} The wrapped version of the supplied function
+*/
+
+function fnBind(fn, context) {
```
Comment (jimmynotjim) [inline cfgov/unprocessed/js/modules/util/fnBind.js]: Minor, add spaces within parens throughout

Comment (sebworks) [inline cfgov/unprocessed/js/modules/util/fnBind.js]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/1866/commits/92ac43e13793816b3f1c1466ccf5f94e0b255dff

File: cfgov/unprocessed/js/modules/util/assign.js (hunk 1 above)
Comment (sebworks) [inline cfgov/unprocessed/js/modules/util/assign.js]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/1866/commits/ce557d2aa816beb344b852015146b1c570d10880

**Comment by jimmynotjim** [comment]
:+1:

**Pull request merged**